README.md

```diff
@@ -264,6 +266,99 @@ It is possible that a standby trying to activate is not able to immediately acqu
 it goes back into standby mode, but if it fails to receive heartbeats from an active server, it will try again to
 acquire the store lock. The interval is random but as of now set to a bit more than a second.
 
+## Static Channels aka Partitioning
```

Comment:
I would recommend sticking with one name here. Probably something along the lines of "Channel Partitions". Maybe even simply "Partitions" and make sure to keep it consistent without the dual options. If we need to make sure people find what they're looking for when they search, we can add it into the help text.